src/BitIntegers.jl

```diff
@@ -331,6 +331,10 @@ flipsign(x::T, y::T) where {T<:XBS} = flipsign_int(x, y)
 flipsign(x::UBS, y::UBS) = flipsign_int(promote(x, y)...) % typeof(x)
 
 
```

Comment:
nitpick: only one line is enough between two functions

Reply:
Fixed.